app-text/zathura/zathura-0.5.2-r5.ebuild

```diff
@@ -50,7 +50,7 @@ PATCHES=(
 src_configure() {
 	local emesonargs=(
 		-Dconvert-icon=disabled
-		-Dmanpages=disabled
+		-Dmanpages=$(usex doc enabled disabled)
```

Comment:
Man pages need to be unconditionally installed: https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0305.

(Always worth checking git history to see why something was done a certain way).